pkg/registry/config.go

```diff
@@ -12,17 +9,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package config
+package registry
 
 const (
-	// TemplateRefreshTimeOut is the template refresh time out for exporting process
-	TemplateRefreshTimeOut uint32 = 1800
-	// TemplateTTL is the template time to live for collecting process
-	TemplateTTL uint32 = TemplateRefreshTimeOut * 3
-	// TemplateSetID is the setID for template record
-	TemplateSetID uint16 = 2
 	// AntreaEnterpriseID is the enterprise ID for Antrea Information Elements
 	AntreaEnterpriseID uint32 = 55829
```

Comment:
You could define these constants pkg/registry/registry.go. Do not need a separate file.